logtracing/trace.go

```diff
@@ -215,6 +229,18 @@ func LogSpan(ctx context.Context, s *span) {
 	l.Info().Log(keyvals...)
 }
 
+func getLogger(ctx context.Context) log.Logger {
```

Comment:
This looks like a reimplementation of `log.ForceContext` https://github.com/theplant/appkit/blob/c3c23cc0e04d38be71ad964ee51ec515f43d20f9/log/context.go#L57-L65

Reply:
> I'd suggest redoing the PR to properly remove the logging changes from the Git history, rather than having a commit to add, and a commit to remove.

Thanks for your review, @bodhi 🙏

I will close this PR and reopen a new one.

> This looks like a reimplementation of log.ForceContext
I will clean up this in the new PR...